### ISSUE TITLE
AvroConsumer/Producer do not support the 'logger' keyword argument

### ISSUE THREAD

**Issue by kontrafiktion** [confluentinc/confluent-kafka-python#657]
Description
===========

The documentation says that it is possible to redirect the logging for the Consumer/Producer by adding `logger=myLogger` as keyword argument, e.g.:

    producer = confluent_kafka.Producer({'bootstrap.servers': 'mybroker.com'}, logger=mylogger)

but the AvroConsumer and the AvroProducer just dismiss any `**kwargs`

How to reproduce
================

add a logger to the AvroProducer and force librdkafka to log something: the output still is sent to stderr

Checklist
=========
Please provide the following information:

 - [x] confluent_kafka.version(): ('1.0.0', 1048576) -- confluent_kafka.libversion(): ('1.0.0', 16777471)
 - [-] Apache Kafka broker version:
 - [-] Client configuration: `{...}`
 - [-] Operating system: any
 - [-] Provide client logs (with `'debug': '..'` as necessary)
 - [-] Provide broker log excerpts
 - [-] Critical issue

**Comment by kontrafiktion** [comment]
If you want, I can create a PR – I just have no idea how to write a test

**Comment by rnpridgeon** [comment]
Place the logger within the config dict like such and it should work 

```
AvroProducer({'bootstrap.servers': 'mybroker.com', 'logger': mylogger}, 
              default_key_schema=key_schema,
              default_value_schema=value_schema)
```

**Comment by danhje** [comment]
Why is this issue still open. Looks to me like it's solved in 1.6.1 and later.
